bootstrap/bootstrap.sh

```diff
@@ -0,0 +1,162 @@
+#                                 Main Code
+# =============================================================================
+# Use language-specific prep script to populate contents of output directory
+if ! mkdir -p "${output_dirpath}"; then
+  echo "Error: Could not create output directory '${output_dirpath}'" >&2
+  exit 1
+fi
+lang_dirpath="${repo_root_dirpath}/${lang}"
+lang_bootstrap_dirpath="${script_dirpath}/${lang}"
+prep_new_repo_script_filepath="${lang_bootstrap_dirpath}/${PREP_NEW_REPO_FILENAME}"
+if ! bash "${prep_new_repo_script_filepath}" "${lang_dirpath}" "${output_dirpath}"; then
+  echo "Error: Failed to prep new repo using script '${prep_new_repo_script_filepath}'" >&2
+  exit 1
+fi
+
+# README file
+output_readme_filepath="${output_dirpath}/${OUTPUT_README_FILENAME}"
+cat <<EOF >"${output_readme_filepath}"
+My Kurtosis Lambda
+=====================
+Welcome to your new Kurtosis Lambda! You can use Example Kurtosis Lambda implementation as a pattern to create your own Kurtosis Lambda.
+
+1. Customize your own Kurtosis Lambda editing the generated files inside the `/path/to/your/code/repos/kurtosis-lambda/impl` folder
+    1. Rename files and objects, if you want, using a name that describes the functionality of your Kurtosis Lambda
+    1. Write the functionality of your Kurtosis Lambda inside the `execute` method
+       1. Define what parameters it will receive and what parameters it will return. Is always a good practice to validate and sanitize the received parameters
+    1. Write the Kurtosis Lambda Configurator for your own Kurtosis Lambda, define which parameters will be used to create and configure the Kurtosis Lambda
+    1. Edit the main file and replace the existing code in order to use your own custom Kurtosis Configurator
+    1. Run `path/to/your/code/repos/scripts/build.sh`, when you finish your Kurtosis Lambda, to update the Docker image
+EOF
+if [ "${?}" -ne 0 ]; then
+  echo "Error: Could not write README file to '${output_readme_filepath}'" >&2
+  exit 1
+fi
+
+#Initialize the new repo as a Git directory, because running the Kurtosis Lambda depends on it
+if ! command -v git &> /dev/null; then
+    echo "Error: Git is required to create a new Kurtosis Lambda repo, but it is not installed" >&2
+    exit 1
+fi
+if ! cd "${output_dirpath}"; then
+    echo "Error: Could not cd to new Kurtosis Lambda repo '${output_dirpath}', which is necessary for initializing it as a Git repo" >&2
+    exit 1
+fi
+if ! git init; then
+    echo "Error: Could not initialize the new repo as a Git repository" >&2
+    exit 1
+fi
+if ! git add .; then
+    echo "Error: Could not stage files in new repo for committing" >&2
+    exit 1
+fi
+if ! git commit -m "Initial commit" > /dev/null; then
+    echo "Error: Could not create initial commit in new repo" >&2
+    exit 1
+fi
+
+#Runs build script
+scripts_dirpath="${output_dirpath}/${SCRIPTS_DIRNAME}"
+bash "${scripts_dirpath}/${BUILD_FILENAME}" "${lambda_image}"
```

Comment:
Same as above - this would require that the user enter the image name every time they want ot run build.sh, which would get tedious very quickly. IMO, during bootstrapping, we should modify the image name inside of the buildscript so that `build.sh` still takes no arguments (the same as Kurt Libs)

Reply:
Also, same as Kurtosis Libs, I think we want to have a "Bootstrap successful! For next steps, follow the instructions in the README at /path/to/new/README"